### PR TITLE
[WIP] DYN-903: Dynamo Dictionary Updates for 2.0

### PR DESCRIPTION
**Pull request by scottmitchell**
**Purpose**

This pull request addresses JIRA task [DYN-903](https://jira.autodesk.com/browse/DYN-903).

**This pull request does**

- [x] Update xml and json to reflect 2.0 library
- [ ] Link new library structure to existing example files and images

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -1 +1 @@
-export const IMAGE_FALSE_AVP_TRUE = true;
+export const IMAGE_FALSE_AVP_TRUE = false;
```
Comment (mjkkirschner) [inline src/constants/index.js]: is this documented somewhere?

**Comment by mjkkirschner** [comment]
@scottmitchell - so a couple questions:
1. what is this PR doing?
2. do we own this repo under dynamo org as well as under here - or is this the only location?
3. what is the relationship to the work you did in the dynamo repo to generate JSON output?
4. where is a sample of that output - will it be committed to this repo? Is that format documented? 
5. Are there tests?

File: src/constants/index.js (hunk 1 above)
Comment (alfarok) [reply src/constants/index.js]: https://github.com/DynamoDS/DynamoDictionary#toggle-avp--example-images

@ScottAMitchell I don't think you want to commit this change, just locally for testing the loading of images

**Comment by alfarok** [comment]
@Racel which version of the Dictionary is deployed?  There is one repo under DynamoDS and one under Modelab and they look pretty different

**Comment by scottmitchell** [comment]
@mjkkirschner 

1. This PR updates the library data in the dictionary: [xml](https://github.com/scottmitchell/DynamoDictionary_React/blob/master/public/data/Revit_Library.xml), [json](https://github.com/scottmitchell/DynamoDictionary_React/blob/master/public/data/Dynamo_Nodes_Documentation.json). 

2. I'll check with @Racel on this.

3. The branches of Dynamo and librarie.js used to generate the new data in this PR can be found here: [Dynamo DYN-903](https://github.com/scottmitchell/Dynamo/tree/DYN-903), [librarie.js DYN-903](https://github.com/scottmitchell/librarie.js/tree/DYN-903)

4. [Here](https://github.com/scottmitchell/librarie.js/blob/DYN-903/src/utility/LibrarieDump.json) is the raw output from the Dynamo/librarie.js dump. The [DictionaryDumpConverter](https://github.com/scottmitchell/librarie.js/blob/DYN-903/src/utility/DictionaryDumpConverter.py) parses that dump to create the xml and json files listed above. No this format is not documented—I will do this.

5. There are no tests currently.

File: src/constants/index.js (hunk 1 above)
Comment (scottmitchell) [reply src/constants/index.js]: @alfarok yep that makes sense

**Comment by Racel** [comment]
@ScottAMitchell  This is the most current repo I know of: https://github.com/DynamoDS/DynamoDictionary_React

It does have AVP in there, so we should remove the example from whatever we deploy publicly.

**Pull request closed without merging**